src/main/fc/rc_adjustments.c

```diff
@@ -413,15 +458,21 @@ void processRcAdjustments(controlRateConfig_t *controlRateConfig)
                 continue;
             }
 
-            applyStepAdjustment(controlRateConfig, adjustmentFunction, delta);
+            newValue = applyStepAdjustment(controlRateConfig, adjustmentFunction, delta);
             pidInitConfig(pidProfile);
         } else if (adjustmentState->config->mode == ADJUSTMENT_MODE_SELECT) {
             const uint16_t rangeWidth = ((2100 - 900) / adjustmentState->config->data.switchPositions);
             const uint8_t position = (constrain(rcData[channelIndex], 900, 2100 - 1) - 900) / rangeWidth;
-            applySelectAdjustment(adjustmentFunction, position);
+            newValue = applySelectAdjustment(adjustmentFunction, position);
         }
+
+#if defined(OSD) && defined(USE_OSD_ADJUSTMENTS)
+        osdShowAdjustment(adjustmentLabels[adjustmentFunction], newValue);
```

Comment:
Should be:
```
if (newValue!=0) {
    osdShowAdjustment(adjustmentLabels[adjustmentFunction], newValue);
}
```

Reply:
Why? I can decrease a pid value down to zero for example.

Reply:
OK, I wasn't sure if `newValue==0` was a valid value. The point is to avoid showing the adjustment if it has not changes, so `newValue` should be initialised to `-1`, and then use:
```
if (newValue!=-1) {
    osdShowAdjustment(adjustmentLabels[adjustmentFunction], newValue);
} 
```

Reply:
done.